packages/portfolio-contract/package.json

```diff
@@ -31,17 +31,21 @@
     "@fast-check/ava": "^2.0.1",
     "ava": "^5.3.0",
     "c8": "^10.1.3",
+    "esbuild": "^0.25.2",
     "ts-blank-space": "^0.6.1"
   },
   "dependencies": {
     "@agoric/cosmic-proto": "workspace:*",
     "@agoric/internal": "workspace:*",
     "@agoric/orchestration": "workspace:*",
     "@agoric/vats": "workspace:*",
+    "@agoric/vow": "workspace:*",
```

Comment:
What are we using the vow package for? oh... `VowShape`. hm. I think that shape really means `M.any()`. I'm hesitant to take on this dependency. But I suppose we already depend on it indirectly. so no big deal.